web-services/query/src/main/java/datawave/webservice/query/runner/QueryExecutorBean.java

```diff
@@ -2064,6 +2066,229 @@ private BaseQueryResponse next(final String id, boolean checkForContentLookup) {
         
         return response;
     }
+
+
+    @GET
+    @Path("/{id}/raw/next")
```

Comment:
I waffled between raw or direct, so I'm open to feedback on how to relay an endpoint that suggests we are not transforming or converting the JSON for the end user.